src/test/java/uk/gov/hmcts/reform/slc/services/FtpAvailabilityCheckerTest.java

```diff
@@ -0,0 +1,59 @@
+package uk.gov.hmcts.reform.slc.services;
+
+import org.assertj.core.api.JUnitSoftAssertions;
+import org.junit.Rule;
+import org.junit.Test;
+
+import java.time.LocalTime;
+
+import static java.time.LocalTime.now;
+import static java.util.Arrays.asList;
+
+public class FtpAvailabilityCheckerTest {
+
+    @Rule
+    public final JUnitSoftAssertions softly = new JUnitSoftAssertions();
+
+    @Test
+    public void should_indicate_whether_ftp_is_available() {
+        test("17:30", "18:30", time(18, 10), false, "During downtime");
+        test("17:30", "18:30", time(22, 15), true, "After downtime");
+        test("17:30", "18:30", time(11, 50), true, "Before downtime");
+        test("17:30", "18:30", time(17, 30), false, "Exactly at downtime start");
+        test("17:30", "18:30", time(18, 30), false, "Exactly at downtime end");
+        test("23:00", "01:00", time(14, 15), true, "Outside of overnight downtime");
+        test("23:00", "01:00", time(23, 59), false, "During overnight downtime");
```

Comment:
Nice coverage. Worth adding the other side of overnight downtime - after midnight.